### PR TITLE
Simple fix for power issue

### PR DESCRIPTION
**Pull request by bluenote10**
simple fix for #5966

### CODE REVIEW

**Comment by bluenote10** [comment]
I discovered that the `tmath` test was actually not active, because the default testament action is "compile only" and no run. It turned out that one of the tests in `tmath` was actually failing, which I have fixed as well.

Thoughts on the testament issue:
- Would it make sense to make "compile + run" the default to avoid missing tests like these?
- It would be nice if testament could run a test without implicitly making an assumption on the expected stdout. I tried making "compile + run" the default and it mainly failed due to the `output == ""` expectation. Also, for a test like this one, copy pasting the output feels unnecessary if we only want to check for the return code.

**Comment by dom96** [comment]
I ran into this problem too, it's easy to miss tests that are supposed to run as only being compiled. I believe I created an issue for this somewhere as well. Not sure what the solution is but something should be done.

File: tests/stdlib/tmath.nim [hunk 1]
```diff
@@ -1,4 +1,15 @@
-import math, random
+discard """
+  action: run
+  output: '''[Suite] random int
```
Comment (Araq) [inline tests/stdlib/tmath.nim]: This is ugly.

Comment (bluenote10) [inline tests/stdlib/tmath.nim]: Yes, that is why I came up with the idea of making the output check optional. And it is also what napalu had to do to fix #5472 in f61bb70.

Comment (Araq) [inline tests/stdlib/tmath.nim]: But instead 'unittest' shouldn't be used. It's a debugging pita.

Comment (bluenote10) [inline tests/stdlib/tmath.nim]: What would you recommend then to do runtime tests? Using only `block`s & `assert`s becomes hard to maintain with a large number of test cases. Going for less runtime behavior tests in general is also not beneficial, and as far as I can see `unittest` is the best tool for that job, isn't it?

**Pull request merged**